publishing.md

```diff
@@ -53,12 +53,12 @@ Please make sure to update the destination of the source code link if you use a
 </span>
 ```
 
-## Distributing changes
+## Changes
 
-If you modify or extend OpenFisca, and if you distribute those changes, you are [legally required](https://choosealicense.com/licenses/agpl-3.0/) to make those changes available to the community. The easiest way to do it is to publish your fork, country package, extension or reform on a source hosting platform such as GitHub, and to [notify](./community.md) the core team of this publication.
```

Comment:
> The addition of “country package” made sense.

Reply:
I added _country package_ back and started the enumeration with _core package_. Do you think it make sense ?